core/src/main/java/com/datastrato/gravitino/storage/relational/mapper/SecurableObjectMapper.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.storage.relational.mapper;
+
+import com.datastrato.gravitino.storage.relational.po.SecurableObjectPO;
+import java.util.List;
+import org.apache.ibatis.annotations.Delete;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+
+/**
+ * A MyBatis Mapper for table meta operation SQLs.
+ *
+ * <p>This interface class is a specification defined by MyBatis. It requires this interface class
+ * to identify the corresponding SQLs for execution. We can write SQLs in an additional XML file, or
+ * write SQLs with annotations in this interface Mapper. See: <a
+ * href="https://mybatis.org/mybatis-3/getting-started.html"></a>
+ */
+public interface SecurableObjectMapper {
+
+  String SECURABLE_OBJECT_TABLE_NAME = "role_meta_securable_object";
+  String ROLE_TABLE_NAME = "role_meta";
+
+  @Insert({
+    "<script>",
+    "INSERT INTO "
+        + SECURABLE_OBJECT_TABLE_NAME
+        + "(role_id, entity_id, type, privilege_names, privilege_conditions, "
+        + " current_version, last_version, deleted_at)"
+        + " VALUES ",
+    "<foreach collection='securableObjects' item='item' separator=','>",
+    "(#{item.roleId},"
+        + " #{item.entityId},"
+        + " #{item.type},"
+        + " #{item.privilegeNames},"
+        + " #{item.privilegeConditions},"
+        + " #{item.currentVersion},"
+        + " #{item.lastVersion},"
+        + " #{item.deletedAt})",
+    "</foreach>",
+    "</script>"
+  })
+  void batchInsertSecurableObjects(
+      @Param("securableObjects") List<SecurableObjectPO> securableObjectPOS);
```

Comment:
Here you mapping relation is:

"one role id -> multiple entity id", is it better change to "one role id -> multiple securable object id -> one combined entity id (or multiple entity id)"

Reply:
Entity id is a unique id. We don't need a combined entity id.

Reply:
What if later on we need to support drop securable from a role, how do we achieve this?

Reply:
We can using full name to get entity id. Using entity id and privilege information to find suitable securable object.